### DISCUSSION TITLE
Question: How to render output into markdown format?

### DISCUSSION THREAD

**Discussion by hanyuancheung** [yuin/goldmark#479] [Q&A]
I'm gonna parse markdown into ast struct, and then modify content in part of some nodes, such as links(Which is because my links are time sensitive).
Eventually, I want to render output into markdown format, how could I do that?

Hope for your answer, thanks!

**Comment by freb** [comment]
A Markdown renderer is not included in this library, unfortunately. But there are others available, like:

https://github.com/teekennedy/goldmark-markdown

See also: https://github.com/yuin/goldmark/discussions/283
